lib/grim/image_magick_processor.rb

```diff
@@ -50,7 +50,7 @@ def prepare_command(pdf, index, path, options)
 
       command = []
       command << @imagemagick_path
-      command << "-resize #{width}"
+      command << "-resize #{width}" if width
```

Comment:
I don't think this will work as is. On line 45 `width` always gets set to something even if the option isn't passed so `width` will always be truthy on line 53.

Might be good to test drive this feature, the current tests can be found in:

* https://github.com/imikay/grim/blob/f2bed5d2905a29c2cc06151024632451b15979c9/spec/lib/grim/image_magick_processor_spec.rb#L46-L50
* https://github.com/imikay/grim/blob/f2bed5d2905a29c2cc06151024632451b15979c9/spec/lib/grim/image_magick_processor_spec.rb#L75-L78

Locally you should be able to run the tests by doing:

```bash
bundle
bundle exec rake test
```